lib/actions/tunnel.ts

```diff
@@ -129,9 +127,11 @@ export const tunnel: CommandDefinition<Args, Options> = {
 				? (options.port as string[])
 				: [options.port as string];
 
-		return inferOrSelectDevice(params.uuid)
+		return getTargetUuid(sdk, params.deviceOrApplication)
 			.then(sdk.models.device.get)
 			.then(device => {
+				logger.logInfo(`Tunnel to ${device.uuid}`);
```

Comment:
I know this isn't technically a change, but maybe something like `Opening a tunnel to ${device.uuid}...` would be better?